src/ducks/components/App.jsx

```diff
@@ -30,7 +30,7 @@ export class App extends Component {
   }
 
   render() {
-    const defaultPart = enabledPages[0]
+    const defaultPart = enabledPages ? enabledPages[0] : 'discover'
```

Comment:
The default should be`['discover', 'myapps']` I think, the same than the current Store config.

Reply:
We are just setting `defaultPart` here.